src/collection-view.js

```diff
@@ -544,10 +544,15 @@ Marionette.CollectionView = Marionette.View.extend({
 
   // Destroy the child views that this collection view
   // is holding on to, if any
-  destroyChildren: function() {
+  destroyChildren: function(checkEmpty) {
```

Comment:
since this is a public api, we should probably accept options instead of just a boolean field.